docs/medical-api/api-reference/patient/create-patient-external-sync.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Create Patient External Sync"
```

Comment:
I think this title is confusing. Maybe `Sync Patient with External System`? Or just `Sync Patient`?